float32 cpu_percent
float32 avg_cpu_percent
float32 mem_usage_mb
uint32 num_threads
float32 cpu_temp